packages/example-codehub/test/acceptance/smoke.test.ts

```diff
@@ -5,10 +5,12 @@
 
 import {CodeHubApplication} from '../../src/codehub-application';
 import {validateApiSpec} from '@loopback/testlab';
+import {RestServer} from '@loopback/rest';
 
-describe('Application', () => {
+describe('Server', () => {
   it('has a valid OpenAPI spec', async () => {
     const app = new CodeHubApplication();
-    await validateApiSpec(app.getApiSpec());
+    const server = await app.get('servers.RestServer') as RestServer;
+    await validateApiSpec(server.getApiSpec());
```

Comment:
Nice. Having transport-specific methods on the Server object makes the design much cleaner, as is avoids the situation where the app becomes a [God object](https://en.wikipedia.org/wiki/God_object) knowing everything about all transports.